### PR TITLE
Add GetInputType to graph runtime

### PR DESCRIPTION
**Pull request by apivovarov**
`GetInputType(int index)` will be used by DLR to get input type. 

`neo-ai/tvm` already has several custom methods in `graph-runtume`  to query Inputs info, such as `GetInputName`, `NumInputs`

Tested in DLR (model-peeker)
```
for (std::string nm: input_names_) {
  int index = tvm_graph_runtime_->GetInputIndex(nm);
  std::string in_type = tvm_graph_runtime_->GetInputType(index);
  printf("=== input: %s, type: %s ===\n", nm.c_str(), in_type.c_str());
}
```
```
# bin/model_peeker ~/workplace/models/mobilenetv2_0.75_cpu
=== input: data, type: float32 ===
backend is tvm
num_inputs = 1
num_weights = 107
num_outputs = 1
input_names: data, 
output shapes: 
[1, 1000, ]
```

### CODE REVIEW

File: src/runtime/graph/graph_runtime.cc [hunk 1]
```diff
@@ -126,6 +126,17 @@ std::string GraphRuntime::GetInputName(int index) const {
...
 }
+/*!
+ * \brief Get the number of the index-th input.
+ * \param index The input index.
+ *
+ * \return The type of the index-th input.
+ */
+std::string GraphRuntime::GetInputType(int index) const {
+  CHECK_LT(static_cast<size_t>(index), input_nodes_.size())
+      << "The index of ouf of range.";
```
Comment (zhiics) [inline src/runtime/graph/graph_runtime.cc]: The index is out of range?

Comment (apivovarov) [reply src/runtime/graph/graph_runtime.cc]: fixed.

File: src/runtime/graph/graph_runtime.cc [hunk 2]
```diff
@@ -123,9 +123,20 @@ int GraphRuntime::NumInputs() const {
  */
 std::string GraphRuntime::GetInputName(int index) const {
   CHECK_LT(static_cast<size_t>(index), input_nodes_.size())
-      << "The index of ouf of range.";
+      << "The index is out of range.";
   return nodes_[input_nodes_[index]].name;
 }
+/*!
+ * \brief Get the number of the index-th input.
```
Comment (zhiics) [inline src/runtime/graph/graph_runtime.cc]: I didn't notice this last time, but the description is not updated. Same to the one in the .h file.

Comment (apivovarov) [reply src/runtime/graph/graph_runtime.cc]: fixed

**Review by zhiics — APPROVED** [review]

**Pull request merged**